openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/CustomThemeConfig.spec.ts

```diff
@@ -85,12 +85,12 @@ test.describe('Custom Theme Config Page', () => {
 
     // Verify the updated theme color
 
-    await page.waitForSelector(
-      '[data-testid="save-btn"] .ant-btn-loading-icon',
-      {
-        state: 'detached',
-      }
+    const spinner = page.locator(
+      '[data-testid="save-btn"] .ant-btn-loading-icon'
     );
+    if (await spinner.isVisible({ timeout: 1000 })) {
+      await spinner.waitFor({ state: 'detached', timeout: 10000 });
+    }
```

Comment:
pls revert, this is a known issue on main and currently being addressed. Don't want to include other changes here.